object_detection/tf_serving_gpu.md

```diff
@@ -0,0 +1,50 @@
+# Serving an object detection model with GPU
+
+## Setup
+
+If you followed previous steps to train the model, skip to deploy [section](#deploy-serving-component).
```

Comment:
The setup steps feel out of place here, since the title is Serving an object detection model with GPU. Could you integrate them with the other markdown files in this example?

Reply:
I think some users might only interested in trying out serving. This provides the steps to do if they skip the training part. And the first line is tell them to skip this if they've done training.

Does it make sense?